examples/connectivity/plot_ica_neurovault.py

```diff
@@ -0,0 +1,260 @@
+"""
+NeuroVault cross-study ICA maps.
+=====================================================================
+
+This example shows how to download statistical maps from
+NeuroVault, label them with NeuroSynth terms,
+and compute ICA components across all the maps.
+
+See :func:`nilearn.datasets.fetch_neurovault` documentation for more details.
+"""
+# Outline
+
+
+""" Code to grab the data from NeuroVault, and compute a map of
+frequency of activation in the brain.
+"""
+# Authors: Chris Filo Gorgolewski, Gael Varoquaux
+# License: BSD
+
+import json
+import io
+import os
+
+import numpy as np
+from joblib import Memory
+from matplotlib import pyplot as plt
+from scipy import stats
+from sklearn.decomposition import FastICA
+from sklearn.feature_extraction import DictVectorizer
+
+import nibabel as nib
+from nilearn import datasets
+from nilearn.datasets.utils import _fetch_files, _get_dataset_dir
+from nilearn.image import resample_img
+from nilearn.input_data import NiftiMasker
+from nilearn.masking import compute_background_mask, _extrapolate_out_mask
+from nilearn.plotting import plot_stat_map
+
+
+mem = Memory('cache', verbose=0)
+
+
+@mem.cache
+def splitext(p):
+    if p.endswith('.nii.gz'):
+        return p[:-7], '.nii.gz'
+    return os.path.splitext(p)
+
+
+@mem.cache
+def resample_and_expand_image(src_nii, target_nii, out_path=None, force=False):
```

Comment:
Taking into account reviews on my PR's, I think ```src_nii``` should be denoted as ```src_img``` same to ```target_nii```.